lgsm/functions/fix_steamcmd.sh

```diff
@@ -79,6 +79,8 @@ elif [ "${shortname}" == "cmw" ]; then
 	fn_fix_steamclient_so "32" "${executabledir}/lib"
 elif [ "${shortname}" == "cs" ]; then
 	fn_fix_steamclient_so "32" "${serverfiles}"
+elif [ "${shortname}" == "ins" ]; then
```

Comment:
this is already merged in to develop. Please remove as not part of the hotfix

Reply:
rebases on master only with a single commit